addons/mail/static/src/components/emoji_category_view/emoji_category_view.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="mail.EmojiCategoryView" owl="1">
+        <span class="o_EmojiCategoryView px-2 py-1 fs-3 o_cursor_pointer" t-att-class="(emojiCategoryView.isHover or emojiCategoryView.isActive) ? 'bg-300' : ''" t-on-click="emojiCategoryView.onClick" t-on-mouseenter="emojiCategoryView.onMouseenter" t-on-mouseleave="emojiCategoryView.onMouseleave" t-attf-class="{{ className }}" t-ref="root">
```

Comment:
minor: the current category looks like it can be clicked even though it does nothing

and its style is the same as the highlight on hover style, which can be a bit confusing

Reply:
Agree, but it's kinda hard to find proper way to distinguish styling, without making the unicode of category hard to read. Darker gray does make some categories very hard to read.